knowledge_base/_Glossary.md

```diff
@@ -0,0 +1,250 @@
+  * [Core](#core)
+  * [Forum](#forum)
+  * [Group](#group)
+  * [Link](#link)
+  * [Member](#member)
+  * [Notification](#notification)
+  * [Page Scope](#page-scope)
+  * [Profile](#profile)
+  * [Reaction](#reaction)
+  * [Poll](#poll)
+  * [Role](#role)
+  * [Sessions](#sessions)
+  * [Sign-In](#sign-in)
+  * [Stake](#stake)
+  * [Subscription](#subscription)
+  * [Template](#template)
+  * [Thread](#thread)
+  * [Topic](#topic)
+  * [User](#user)
+  * [Wallet](#wallet)
+  * [Votes](#votes)
+- [Hexagonal Architecture](#hexagonal-architecture)
+  * [Port](#port)
+  * [Adapter](#adapter)
+- [Praxis](#praxis)
+  * [Blocked](#blocked)
+  * [Fast-Follow](#fast-follow)
+  * [Issues, Tickets, & Stories](#issues-tickets--stories)
+- [Change Log](#change-log)
+
+## Chain Concepts
+
+### Chain Base
+
+An architectural layer of blockchain atop which NETWORK protocols are built.
+
+### Chain Entity
+
+A Chain Entity is a backend object, stored in the ChainEntities table, which consists of chain event bundles and values received from a smart contract.
+
+### Chain Node
+
+An endpoint, referring to external RPC servers, used to fetch updates and query data from blockchains.
+
+### Delegate Contract
+
+A delegate contract is a smart contract that contains the functions, events, and all business logic for a proxy contract.
+
+### Module
+
+Some CHAINS use modules rather than contracts. Modules are used predominantly within the Cosmos and Substrate ecosystems.
+
+### Network
+
+An ecosystem of CHAIN NODES, or blockchain, built atop a CHAIN BASE.
+
+## Common App
+
+### Bridge
+
+The Product Team uses "bridge" and "bridging" language, in user-facing comms, to refer to implementation-agnostic, cross-platform integrations such as [Discobot](./Discobot.md) and Farcaster.
+
+### Comment
+
+A short Markdown text object responding to a THREAD, which can be REACTED to.
+
+See [models/comment.ts](../libs/model/src/models/comment.ts).
+
+### Community
+
+<!-- TODO: Review & update in light of new permissioning system; blocked pending meeting with Product. -->
+
+A community is a space on Common, joinable by PROFILES, that owns metadata and is defined by a set of ROLES and PERMISSIONS for participation.
+
+See [models/community.ts](../libs/model/src/models/community.ts)
+
+### Core
+
+A set of features considered central to Common, which interface out with first-party apps. As of 240123, the core consists of a SIGN-IN system, a NOTIFICATION system, and a GROUP system.
+
+### Forum
+
+The Common forum is Common's flagship first-party app. Its primary organizational primitive is the COMMUNITY. The forum app provides UI for off-chain discussions, and for interacting with community-associated chains.
+
+### Group
+
+<!-- TODO: Review & update in light of new permissioning system; blocked pending meeting with Product. -->
+
+Groups are used by community admins to handle permissioning across the forum, using associated membership objects that LINK ADDRESSES to Groups.
```

Comment:
Better phrasing is "Addresses have Memberships to Groups".

Reply:
I can def clear up the meaning but—

In a situation with a join or junction table ("Memberships") why privilege Addresses' ownership? Isn't it equally true that Groups have Member Addresses?

Reply:
From a database perspective it's equally true that Groups have Member Addresses, but emphasizing addresses makes more sense from the product perspective, where users will ask "What Addresses are in the Group?" which semantically emphasizes the Addresses's quality and not the Group's quality.